core/server/services/members/api.js

```diff
@@ -25,11 +25,22 @@ function updateMember(member, newData) {
     });
 }
 
-function getMember(member) {
-    return models.Member.findOne(member, {
-        require: true
-    }).then((member) => {
-        return member.toJSON();
+function getMember(data, options) {
+    options = options || {};
+    return models.Member.findOne(data).then((model) => {
```

Comment:
Removing `require: true` from here breaks the contract it would have held with the members module, previously we would reject if the member was not found - is there a reason for changing this?

Reply:
Yeah missed adding it [here](https://github.com/TryGhost/Ghost/pull/10435/files#diff-85e599bebda917a3366f5c838f86329bR12) before, updated. Basically the option can be added where we need it, like the contract right now is in `requestPasswordReset` method setup, but doesn't need to be here as generic option. Updated now in the method so it breaks the same way in case of member not found

Reply:
IMO forcing the members module to pass `require: true` create a coupling between members and the models we use - but not a big deal can be fixed later :relaxed:

Reply:
Ah yes, guess we could just do the equivalent with checking for null value in `requestPasswordReset` and throwing an error, but yeah we can iterate on that later. :)